src/style.css

```diff
@@ -1 +1,58 @@
+/* fons */
```

Comment:
The comment 'fons' seems to be a typo. Consider correcting it to 'fonts' for clarity.